lib/inline_svg/action_view/helpers.rb

```diff
@@ -4,7 +4,29 @@
 module InlineSvg
   module ActionView
     module Helpers
+      def inline_svg_tag(filename, transform_params={})
+        with_asset_finder(nil) do
```

Comment:
If I'm reading this (and `with_asset_finder`) correctly, passing `nil` here might be a problem because it could override a user-configured asset finder: https://github.com/jamesmartin/inline_svg/blob/e31b68e265dedc31fa7e77d5a4d1e1ec246401ee/lib/inline_svg/railtie.rb#L16-L20

What if we passed the _current_ asset finder as the argument here? 🤔 

```suggestion
        with_asset_finder(InlineSvg.configuration.asset_finder) do
```